app/controllers/decks_controller.rb

```diff
@@ -88,20 +88,22 @@ def create
   end
 
   def contributors
-    @deck = Deck.where(id: params[:id]).includes(:cards, :card_edits).first
-    # @deck = Deck.where(id: 1).includes(:cards, :card_edits).first
-    cards = @deck.cards
-    edits = @deck.card_edits
-    contributors = Deck.contributions(cards, edits)
-    user_ids = contributors[:authors].keys + contributors[:editors].keys
-    users = User.where(id: user_ids)
-
-    response = users.collect do |user|
+    # grab the deck and include all of its cards ('created cards') and card_edits (recomendations to cards)
```

Comment:
@natomato I added some comments and changed the attribute names on the response from created and edited to cards_created_count and cards_edited_count.